backend/code/archipelag/templates/base.html

```diff
@@ -1,23 +1,23 @@
-<!DOCTYPE html> {% load bootstrap3 %}
+<!DOCTYPE html>
 <html lang="en">
 
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="styles.css">
-    {% bootstrap_css %}
+
 </head>
 
 <body>
     <nav class="navbar navbar-default navbar-static-top">
         <div class="container-fluid">
             <ul class="nav navbar-nav navbar-left">
-                <li><a href="#">Dostępne punkty:</a></li>
+                <li><a href="#">Witaj {{ request.user }} masz: {{ request.user.ngouser.coins }} punktów</a></li>
```

Comment:
At this point, you can think about translations (it is not needed, but it is a good way to practice).
https://docs.djangoproject.com/en/1.11/topics/i18n/translation/